packages/coinstac-ui/app/config.js

```diff
@@ -38,6 +38,13 @@ const conf = convict({
     noURLPrefix: true
   },
   logFile: 'coinstac-log.json',
+  logLocations: {
```

Comment:
Not sure that we _need_ configurable log locations. Might be needlessly complex. We can address at another time.